relays/exchange-relay/src/exchange_loop.rs

```diff
@@ -22,18 +22,13 @@ use crate::exchange::{
 };
 use crate::exchange_loop_metrics::ExchangeLoopMetrics;
 
-use backoff::backoff::Backoff;
 use futures::{future::FutureExt, select};
 use num_traits::One;
 use relay_utils::{
 	metrics::{start as metrics_start, GlobalMetrics, MetricsParams},
-	retry_backoff,
```

Comment:
Actually backoff was still in headers+messages loops. Here it was removed because exchage loop is not like the other loops - it isn't fully async. It is async from Rust pov, but it has not simultaneous requests to source/target nodes and any internal buffers, so it's more like linear. But I've returned backoff for non-connection errors. I agree that removal was unjustified here.

Please note that I'm not using backoff mechanism for connection-related errors in all loops intentionally. That's because connection-related errors are not creating (at least it was assumed in the design) additional load on client node. So imagine your network is dropped for 10m. In this case backoff mechanism may take another 5m to retry connection. This all will lead in total >15m delay of all bridge-related applications. Delay 10m10s is obviously better than that, given that it doesn't create additional load on target node.